### ISSUE TITLE
Discovery: Drupal inline form validation

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#13115]
## Description
One of the updates we are looking to make to the editorial interface for images is to add form validation on the alt text field if an editor tries to save an image with a filename in the field.  We are thinking that this error should be presented inline.  We need to investigate available functionality and options within Drupal for providing inline form validation errors. It is also important to note that we are intending for this error to be customizable - to make sure we can provide a clear message to the editor as to what the issue is and how they can resolve it.

### Additional resources

- https://antistatique.net/en/blog/drupal-inline-messages-validation-in-forms
https://www.drupal.org/docs/8/core/modules/inline-form-errors/inline-form-errors-module-overview 
- https://antistatique.net/en/blog/drupal-inline-messages-validation-in-forms
- [Inline Form Errors module overview](https://www.drupal.org/docs/8/core/modules/inline-form-errors/inline-form-errors-module-overview)

## Acceptance Criteria
- [ ] Confirm Drupal is not currently providing inline form validation
- [ ] Research/investigate options for adding inline form validation to the CMS
- [ ] Provide recommendations for adding this functionality
- [ ] Provide LOE estimate for adding this functionality

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by tonytaylor** [comment]
Document detailing form validation review & solution candidate can be found [here](https://docs.google.com/document/d/1GnefY2XvxUMljZcU-nr4wdwsit_Vzu0yaFt99nExb30/edit?usp=sharing).
Document in Word format can be downloaded at [VA.gov CMS_ Form Validation Editor Experience (EX).docx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/11205085/VA.gov.CMS_.Form.Validation.Editor.Experience.EX.docx)
